frontend/src/views/Lobby.tsx

```diff
@@ -331,6 +333,25 @@ function LobbyPage() {
       });
   };
 
+  const updateSize = () => {
```

Comment:
This comment isn't specifically related to this line, but install the [Visual Studio Code ESLint Extension](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint) and address the errors / warnings that come up! These include a max line length of 100, using single-quotes for strings, and using expected property shorthand. Let me know if you have any troubles with this - this is the same extension that I have, but if you run into issues or don't see these warnings just message me.

Reply:
I installed the extension, and no warnings or issues came up. or at least I didn't see any. Where should I look for the warnings?

Reply:
Hmm, they should simply be red underlines on relevant parts of the code - maybe you have to close and reopen VSCode? Otherwise I'm not sure, we can investigate on the daily call tomorrow.